labs.yml

```diff
@@ -132,6 +132,8 @@ commands:
         description: Subscription to scan storage account in
       - name: aws-profile
         description: AWS Profile to use for authentication
+      - name: run-as-collection
```

Comment:
Note that boolean flags are not supported by the Databricks cli: https://github.com/databrickslabs/ucx/pull/1925#discussion_r1688198505

Reply:
the current logic still works by passing the value as True. as discussed , will include it in the next PR